### PR TITLE
Fix travis config file

### PR DESCRIPTION
**Pull request by IanDelMar**
Fixes issues arising from:
* Directory changed due to version update in #1040.
* Ruleset xml-file was renamed in #995.
* PHP Compatibility sniffs were added in #1005.

### CODE REVIEW

**Comment by IanDelMar** [comment]
Getting closer to a working travis config :-D

**Comment by IanDelMar** [comment]
@UnderstrapFramework the configuration should be fixed now (tested with a copy of this repository). Please merge this PR so that builds no longer fail due to the misconfiguration.

File: .travis.yml [hunk 1]
```diff
@@ -49,14 +55,14 @@ before_script:
...
-    - mkdir php-codesniffer && curl -L https://github.com/squizlabs/PHP_CodeSniffer/archive/3.5.0.tar.gz | tar xz --strip-components=1 -C php-codesniffer
+    - git clone -b master --depth 1 https://github.com/squizlabs/PHP_CodeSniffer.git $PHPCS_DIR
     # Install WordPress Coding Standards.
-    - mkdir wordpress-coding-standards && curl -L https://github.com/WordPress/WordPress-Coding-Standards/archive/2.1.1.tar.gz | tar xz --strip-components=1 -C wordpress-coding-standards
-    # Hop into CodeSniffer directory.
-    - cd php-codesniffer
-    # Set install path for WordPress Coding Standards
+    - git clone -b master --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git $WPCS_DIR
+    # Install PHP Compatibility sniffs.
+    - git clone -b master --depth 1 https://github.com/wimg/PHPCompatibility.git $PHPCOMPAT_DIR
```
Comment (jrfnl) [inline .travis.yml]: Hmm... why are you using an abandoned repo ? And even, why use the generic `PHPCompatibility` standard when this is a WP repo and would be better off using [`PHPCompatibilityWP`](https://github.com/PHPCompatibility/PHPCompatibilityWP) ?

Comment (IanDelMar) [reply .travis.yml]: You're right should have been https://github.com/PHPCompatibility/PHPCompatibility/. As stated in the description, I tried to make the travis config work again after months of failing builds due to the things changed in the mentioned three PRs (one of them introducing PHPCompatibility). It's still not working anyway. I'll check it out again this weekend.

Comment (pattonwebz) [reply .travis.yml]: Hey @jrfnl, don't really have any good input here but wanted to say hi XD

Comment (jrfnl) [reply .travis.yml]: @IanDelMar Iterating step by step towards a final solution is not a bad thing :wink: Hopefully I've given you some inspiration for the next iteration in this case.

@pattonwebz :wave:

Comment (jrfnl) [reply .travis.yml]: @IanDelMar Feel free to mention me in a next PR if you want my feedback & I'll try and look it over if I have time.

Comment (IanDelMar) [reply .travis.yml]: Thank you, I’ll do that. But may I ask why you don’t make a PR?

Comment (jrfnl) [reply .travis.yml]: @IanDelMar I understand where you're coming from but:
1. I don't use and never have used Understrap.
2. I kind of maintain enough projects as it is, including PHPCompatibility and WPCS....

Comment (IanDelMar) [reply .travis.yml:62]: > @IanDelMar Feel free to mention me in a next PR if you want my feedback & I'll try and look it over if I have time.

@jrfnl May I ask you to have a look at #1165? I'd appreciate your feedback!

Comment (jrfnl) [reply .travis.yml:62]: Done

**Pull request merged**